README.md

```diff
@@ -1,472 +1,302 @@
 # My OSX Configuration
 
+I like minimalist setup.
+
 ## Cleaning your mac
 
-It is good to factory reset your mac often, to be aware of everything you have installed and to remove unwanted clutter.
+It is good to factory reset your mac often to be aware of everything you have installed and to remove unwanted clutter.
 
 The steps for cleaning your mac are as follows:
 
-+ Back up what you need
-+ Boot your mac in recovery mode (Restart while holding `Cmd + R`)
-+ Select `Disk Utility`, then click `Continue`
-+ Click `Internal > Untitled`, then `Erase`
-+ Click `Erase` in the prompt
-+ Once this has completed, click `Done`
-+ Close the window by clicking the red cross in the top left
-+ Click `Reinstall macOS`, then `Continue`
-
-## Applications to download
-
-+ [**Googe Chrome**](https://github.com/samhstn/my-config/tree/master#googe-chrome) - My browser choice
-
-+ [**Karabiner-Elements**](https://github.com/samhstn/My-Config/tree/master#karabiner-elements) - For key remappings
-
-+ [**Spectacle**](https://github.com/samhstn/My-Config/tree/master#spectacle) - For quick setting of application window sizes
-
-+ [**Iterm2**](https://github.com/samhstn/My-Config/tree/master#iterm2) - For a better terminal
-
-+ [**Docker**](https://github.com/samhstn/My-Config/tree/master#docker) - For configuring reliable deployments
-
-## Programs to install
-
-+ [**Zsh**](https://github.com/samhstn/My-Config/tree/master#zsh)
-
-+ [**Vim**](https://github.com/samhstn/My-Config/tree/master#vim)
++ Back up what you need.
++ Boot your mac in recovery mode (Restart while holding `Cmd + R`).
++ In the `Utility` menu, select `Disk Utility`.
++ Click `Erase` on all `Internal` drives (where permitted).
++ Once this has completed, click `Done`, then close the window to get back to the original menu.
++ Click `Reinstall macOS`, then `Continue`.
 
-+ [**Homebrew**](https://github.com/samhstn/My-Config/tree/master#homebrew)
+## Applications and programs to install
```

Comment:
great list of tools @samhstn! a few I haven't tried. adding to list for Saturday experimentation. 👍